src/StreamJsonRpc/JsonMessageFormatter.cs

```diff
@@ -25,13 +26,23 @@ namespace StreamJsonRpc
     /// <summary>
     /// Uses Newtonsoft.Json serialization to serialize <see cref="JsonRpcMessage"/> as JSON (text).
     /// </summary>
-    public class JsonMessageFormatter : IJsonRpcMessageTextFormatter
+    public class JsonMessageFormatter : IJsonRpcMessageTextFormatter, IJsonRpcInstanceContainer
     {
+        /// <summary>
+        /// Event set when $/progress notification is processed.
+        /// </summary>
+        //public static ManualResetEvent ProgressReceivedSignal = new ManualResetEvent(false);
+
```

Comment:
Why is this commented out?

Reply:
This looks like the wrong update again. I removed this latest one.